Packs/WALLIXBastion/Integrations/WAB/WAB.py

```diff
@@ -1507,2565 +1446,296 @@ def get_sessions_request(
-        app_account_put_certificate_validity,
-        app_account_put_checkout_policy,
-        app_account_put_description,
-        app_account_put_onboard_status,
-    )
-    command_results = CommandResults(
-        outputs_prefix="WAB",
-        outputs_key_field="",
-        outputs=response,
-        raw_response=response,
-    )
-
-    return command_results
-
-
-def edit_account_on_local_domain_of_device_command(
-    client: Client, args: Dict[str, Any]
-) -> CommandResults:
-    device_id = str(args.get("device_id", ""))
-    domain_id = str(args.get("domain_id", ""))
-    account_id = str(args.get("account_id", ""))
-    force = argToBoolean(args.get("force", False))
-    device_account_put_account_login = str(
-        args.get("device_account_put_account_login", "")
-    )
-    device_account_put_account_name = str(
-        args.get("device_account_put_account_name", "")
-    )
-    device_account_put_auto_change_password = argToBoolean(
-        args.get("device_account_put_auto_change_password", False)
-    )
-    device_account_put_auto_change_ssh_key = argToBoolean(
-        args.get("device_account_put_auto_change_ssh_key", False)
-    )
-    device_account_put_can_edit_certificate_validity = argToBoolean(
-        args.get("device_account_put_can_edit_certificate_validity", False)
-    )
-    device_account_put_certificate_validity = str(
-        args.get("device_account_put_certificate_validity", "")
-    )
-    device_account_put_checkout_policy = str(
-        args.get("device_account_put_checkout_policy", "")
-    )
-    device_account_put_description = str(args.get("device_account_put_description", ""))
-    device_account_put_onboard_status = str(
-        args.get("device_account_put_onboard_status", "")
-    )
-    device_account_put_services = argToList(args.get("device_account_put_services", []))
-
-    response = client.edit_account_on_local_domain_of_device_request(
-        device_id,
-        domain_id,
-        account_id,
-        force,
-        device_account_put_account_login,
-        device_account_put_account_name,
-        device_account_put_auto_change_password,
-        device_account_put_auto_change_ssh_key,
-        device_account_put_can_edit_certificate_validity,
-        device_account_put_certificate_validity,
-        device_account_put_checkout_policy,
-        device_account_put_description,
-        device_account_put_onboard_status,
-        device_account_put_services,
-    )
-    command_results = CommandResults(
-        outputs_prefix="WAB",
-        outputs_key_field="",
-        outputs=response,
-        raw_response=response,
-    )
-
-    return command_results
-
-
-def edit_application_command(client: Client, args: Dict[str, Any]) -> CommandResults:
-    application_id = str(args.get("application_id", ""))
-    force = argToBoolean(args.get("force", False))
-    application_put__meters = str(args.get("application_put__meters", ""))
```

Comment:
Totally agree on the principle, but in this case the application_put__meters was an invalid argument produced by the demisto openapi codegen tool that I missed when testing the integration. Therefore, it would have been ignored when executing the command. Removing it will not change the behaviour of the command. Can we make an exception in this case?

Reply:
I understand, how about we handle this argument in the code (like my suggestion), and mark it as hidden in the yml? That way we can avoid errors if any user used this (even if it wasn't functional), but not have it visible.